src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Serializer.cs

```diff
@@ -659,7 +659,10 @@ protected virtual Saml2AuthenticationContext ReadAuthenticationContext(XmlDictio
                     reader.ReadStartElement(Saml2Constants.Elements.AuthnContextDeclRef, Saml2Constants.Namespace);
 
                 // Now we have enough data to create the object
-                var authnContext = new Saml2AuthenticationContext(classRef, declRef);
+                var authnContext = new Saml2AuthenticationContext(classRef);
```

Comment:
This is to keep the behavior the same. Previously, classRef was always passed in and always validated as non-null. declRef was allowed to be null.

Reply:
Makes sense.